src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -65,6 +65,8 @@ export default class MeetingContainer extends Component {
       teamMembers !== this.props.memberSub.data.teamMembers ||
       team.activeFacilitator !== oldTeam.activeFacilitator ||
       user.id !== this.props.user.id) {
+      // We've changed teams, reset our container's state:
+      this.resetContainer();
```

Comment:
Well a debugger call in componentWillUnmount() certainly doesn't get tripped when typing in a new URL. But that may be a false negative. Going to poke around more carefully now...